src/components/enterprise-user-subsidy/data/hooks/hooks.js

```diff
@@ -251,7 +252,7 @@ const getRedeemablePoliciesData = async ({ queryKey }) => {
   const userID = queryKey[2];
   try {
     const response = await fetchRedeemableLearnerCreditPolicies(enterpriseId, userID);
-    return camelCaseObject(response.data);
+    return camelCaseObject(transformRedeemablePoliciesData(response.data));
```

Comment:
Is it necessary to make an extra functional call there?  i think some extra iterations or transformations make it a bit slow. Can we not achieve this in `getTransformedAllocatedAssignments` ?

Reply:
No, since we don't have `subsidy_expiration_date` in assignments. The whole point of this method is to attach `subsidy_expiration_date` with all learner_content_assignments so that we can use it later.